oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticPropertyIndexTest.java

```diff
@@ -154,4 +173,51 @@ public void propertyExistenceQuery() throws Exception {
                 Arrays.asList("/test/a", "/test/b")));
     }
 
+
+    @Test
+    public void testSim() throws Exception {
```

Comment:
testSimilarity

Reply:
+1 on changing the name. I tend to not include `test` since we already have the annotation telling us this is a test. I think that something like `vectorSimilarity` is nicer.